src/TIN.php

```diff
@@ -109,6 +109,16 @@ public static function fromSlug(string $slug): TIN
         return new self($slug);
     }
 
+    public static function isCountrySupported(string $countryCode): bool
+    {
+        foreach (self::$algorithms as $algorithm) {
+            if (true === $algorithm::supports($countryCode)) {
```

Comment:
Shouldn't we uppercase the `$countryCode` by default?

Reply:
I thought about that as well, but the supports function automatically does that for us:
`
    final public static function supports(string $country): bool
    {
        return strtoupper($country) === strtoupper(static::COUNTRYCODE);
    }
`

Reply:
Oh fair enough.